pandas/tests/series/test_operators.py

```diff
@@ -962,6 +962,34 @@ def test_timedelta64_ops_nat(self):
 
 
 class TestDatetimeSeriesArithmetic(object):
+    @pytest.mark.parametrize('box_cls, assert_func', [(Series,
```

Comment:
this formatting is a bit atypical, put a more like

```
@pytest.mark.parametrize(
    'box, assert_func', 
    [(Series, tm.assert_series_equal),
      (pd.Index, tm.assert_index_equal)])
```
is much better, also rename ``box_cls`` -> ``box`` (consistent with elsewhere)